odl_video/settings.py

```diff
@@ -73,11 +75,31 @@ def get_var(name, default):
     )
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```

Comment:
We should change this to use an environment variable. Do we have an issue for this?